api/test/service/store.test.js

```diff
@@ -63,10 +64,16 @@ test.afterEach(async t => {
 })
 
 test('store/add returns signed url for uploading', async (t) => {
+  const { tableName, bucketName } = await prepareResources(t.context.dynamoClient, t.context.s3Client)
+
   const uploadService = await Signer.generate()
   const alice = await Signer.generate()
   const { proof, spaceDid } = await createSpace(alice)
-  const connection = await getClientConnection(uploadService, t.context)
+  const connection = await getClientConnection(uploadService, t.context, {
```

Comment:
feels odd to have a different fn signature for this in tests.
```suggestion
  const connection = await getClientConnection(uploadService, { ...t.context, tableName, bucketName })
```